x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_log_expanded_tray.tsx

```diff
@@ -234,7 +237,17 @@ export const ActionsLogExpandedTray = memo<{
       },
       {
         title: OUTPUT_MESSAGES.expandSection.hostname,
-        description: map(hosts, (host) => host.name).join(', ') || emptyValue,
+        description:
+          Object.values(hosts)
```

Comment:
As discussed locally, this is the reason why `results.cy.ts` breaks, `hosts` is undefined. 
So I'd suggest either verify why it's undefined, or just skip using `Object.values()` and use `reduce` from lodash.

Reply:
done 0222ac6ffe49c2fafb0bd036648d5be00135ff0d